cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,16 @@
+/// <reference types="cypress" />
+const { generateUser } = require('../support/generate');
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
 
+    cy.visit('/');
+    cy.contains('Sign up').click();
+    cy.getByPlaceholder('Username').type(user.username);
+    cy.getByPlaceholder('Email').type(user.email);
+    cy.getByPlaceholder('Password').type(user.password);
+    cy.get('.btn').click();
```

Comment:
The selector '.btn' is too generic. It would be better to use a more specific selector to avoid interacting with other unintended elements.